src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,89 @@
+.menu {
+  background-color: $colorRed;
```

Comment:
Consider using a more descriptive variable name for the background color, e.g., $menuBackgroundColor.